nrcemt/qeels/gui/main_window.py

```diff
@@ -138,6 +138,7 @@ def __init__(self):
         self.material_list = tk.Listbox(
             list_frame,
             listvariable=dropdown_var,
+            selectmode="select",
```

Comment:
does this fix anything?

Reply:
I don't think select is a valid value for this

Reply:
tk tcl seems indicate, `single, browse, multiple, or extended` are valid options

Reply:
if it doesn't fix it, just remove it for now, because it may cause a merge conflict with your next pr

Reply:
but again don't spend too much time fussing with listbox, we can just bail

Reply:
I was just playing around with it, I’ll remove it.